code/modules/mob/living/living_say.dm

```diff
@@ -182,6 +182,9 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 		var/list/hsp = handle_speech_problems(message_pieces, verb)
 		verb = hsp["verb"]
 
+	if(cannot_speak_loudly())
+		return whisper(message)
+
```

Comment:
need you to double-check that this properly logs everything since you're doing a return before create_log() statements. Also I'm not 100% if this loops back around to this proc again or not but just wanted to check.

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/18598676/5ea1e299-d4a5-4a52-9fc5-9814237f56f6)